insights/parsers/tuned_conf.py

```diff
@@ -0,0 +1,92 @@
+"""
+This module contains following parser:
```

Comment:
You may want to add a title for this file:
```
'''
Title XXX
=========
'''
```

Reply:
Done